install-php-extensions

```diff
@@ -1377,6 +1376,23 @@ installPECLModule() {
 	esac
 }
 
+installPickle() {
+	if ! test "$PHP_MAJMIN_VERSION" = "800"; then
```

Comment:
What about moving this PECL related stuff below the `if ! stringInList "$PHP_MODULE_TO_INSTALL" "$BUNDLED_MODULES"; then` line? That way, we'd run `pecl channel-update pecl.php.net || true` only if we'll use PECL.

Also, what about replacing `if ! test "$PHP_MAJMIN_VERSION" = "800"; then` with `if test $PHP_MAJMIN_VERSION -lt 800; then` ?

Reply:
Yes, makes sense to move the whole check for php 8 inside the loop.